.github/workflows/container-tests.yml

```diff
@@ -84,6 +84,14 @@ jobs:
         with:
           ref: "refs/pull/${{ github.event.issue.number }}/head"
 
+      - name: Obtain envvars from PR's comment
+        id: user_defined
+        shell: bash
+        run : |
+          _comment=${{ github.event.comment.body }}
+          _vars=$(echo $_comment | grep -Eo '\[ENV:.*\]' | cut -d':' -f2 | tr -d ']')
```

Comment:
Do we allow more then one [ENV:<variable>]?

Reply:
Yeah, I should have documented it somewhere.
the [ENV:] can be used only once, but you can specify more env vars there. Example: [ENV:DEBUG=1;FAIL_FAST=3;TEST=test]

Reply:
What about if user specify `[ENV:DEBUG;FAIL_3;TEST]`?

Reply:
That would not work. Should it be working? IMO not.

Reply:
And for the comment about documentation of semantics of the value -- we can either have a note how to define an arbitrary variable in the README.md in the root of this repo.. or as a comment here in the yaml?

Reply:
I vote for the comment in yaml.